### PR TITLE
Merge main to feature/rds-phase-3

### PR DESCRIPTION
**Pull request by vpnguye2**
**Summary:** Summary of changes
Merge main into feature branch rds-phase-3

## PR Checklist

- [ ] Update CHANGELOG
- [X] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [X] Integration tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -121,6 +146,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
     - `tasks/hyrax-metadata-updates`
     - `tasks/lzards-backup`
     - `tasks/sync-granule`
+- **CUMULUS-2886**
```
Comment (Jkovarik) [inline CHANGELOG.md:149]: Was this added post release on main?

File: packages/api/endpoints/reconciliation-reports.js [hunk 2]
```diff
@@ -80,8 +81,8 @@ async function getReport(req, res) {
       // estimated payload size, add extra
       const estimatedPayloadSize = presignedS3Url.length + reportSize + 50;
       if (
-        estimatedPayloadSize >
-        (process.env.maxResponsePayloadSizeBytes || maxResponsePayloadSizeBytes)
+        estimatedPayloadSize
+          > (process.env.maxResponsePayloadSizeBytes || maxResponsePayloadSizeBytes)
```
Comment (Jkovarik) [inline packages/api/endpoints/reconciliation-reports.js:85]: Not this PR, but noting to make sure this was an intentional change

**Review by Jkovarik — APPROVED** [review]
This looks good to me, in that it's representative of main's changes.     Integration tests are failing, please as for re-review if change commits are required to fix them.

File: packages/api/endpoints/reconciliation-reports.js (hunk 2 above)
Comment (vpnguye2) [reply packages/api/endpoints/reconciliation-reports.js:85]: Made a note comment here in the next PR for ingest load test

**Review by jennyhliu — APPROVED** [review]
LGTM

File: packages/es-client/tests/test-execution-upsert.js [hunk 3]
```diff
@@ -31,7 +31,7 @@ test.after.always(async (t) => {
   await cleanupTestIndex(t.context);
 });
 
-test.serial('upsertExecution writes new "running" execution', async (t) => {
+test.only('upsertExecution writes new "running" execution', async (t) => {
```
[the remark this answers is not in the document]
Comment (vpnguye2) [reply packages/es-client/tests/test-execution-upsert.js:34]: Awesome. I will do the merge now. Thanks Jenny

**Pull request merged**